etc/conan/profiles/release-linux

```diff
@@ -0,0 +1,12 @@
+[settings]
+os=Linux
+arch=x86_64
+build_type=Release
+compiler=gcc
+compiler.cppstd=gnu20
```

Comment:
We generally prefer ISO standard over GNU.
```suggestion
compiler.cppstd=20
```

Reply:
Updated this one based on your feedback.